graders/python/python_autograder/pltest.py

```diff
@@ -0,0 +1,127 @@
+from unittest import TestLoader, TestSuite
+import json
+import traceback
+import os
+from collections import defaultdict
+from pl_result import PrairieTestResult
+
+
+"""
+The main python entrypoint for the autograder framework.
+Loads and executes test cases.
+"""
+
+
+def add_files(results):
+    BASE_DIR = "/grade/run/"
+
+    for test in results:
+        test["files"] = test.get("files", [])
+        image_fname = BASE_DIR + "image_" + test["name"] + ".png"
+        if os.path.exists(image_fname):
+            with open(image_fname, 'r') as content_file:
+                imgsrc = content_file.read()
+            if not 'images' in test:
+                test['images'] = []
+            test['images'].append(imgsrc)
+            os.remove(image_fname)
+        feedback_fname = BASE_DIR + "feedback_" + test["filename"] + ".txt"
+        if os.path.exists(feedback_fname):
+            with open(feedback_fname, 'r', encoding='utf-8') as content_file:
+                text_feedback = content_file.read()
+            test['message'] = text_feedback
+            os.remove(feedback_fname)
+
+
+if __name__ == '__main__':
+    try:
+        from filenames.test import Test as test_case
+
+        with open('filenames/output-fname.txt', 'r') as f:
+            output_fname = f.read()
+        os.remove('filenames/output-fname.txt')
```

Comment:
Add a comment about why we remove?

Reply:
Not addressing just this comment, but the "security" in general.  This was my patched-together idea of preventing students from cheating in the easiest ways I could think of.  The whole concept could probably use a review by someone who has more experience hardening things.